src/main/resources/drivers/mapping/chromeMapping.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
format xmls with 2 space tab. Use for example https://jsonformatter.curiousconcept.com/